en/starter/settingup.md

```diff
@@ -24,7 +24,7 @@ redirect_from: "/starter/setting up.html"
 <hr>
 ## Ubuntu Linux
 
-Kitura is tested on Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS.
+Kitura is tested on Ubuntu 14.04 LTS.
```

Comment:
@shmuelk I agree that our developers should not exclusively depend on Travis-CI to ensure that their code works on Linux… but I have seen that. :-/ That’s why I was wondering how accurate it is to say that we test and support all of our Swift packages on those three flavors of Ubuntu. The only Ubuntu version that is used for continuous integration and continuous tests is Ubuntu 14.04 (unless you use Docker in Travis-CI with the corresponding image…). 

It then seems that the Kitura team does not have yet a continuous integration pipeline for building and testing all of the Kitura packages on Ubuntu 16.04… and if developers are doing this manually, we do not know how often that manual process occurs (probably less frequent than we think) and how many of our Kitura packages (all of them, or just a few?) are covered by this manual process. Is that a fair statement to say or did I miss something? Should we then only keep Ubuntu 14.04 in this page until the work you mentioned regarding Ubuntu 16.04 and Travis is completed and used by all the Kitura packages?